vocabs.py

```diff
@@ -22,16 +22,24 @@ def get_dataId2word(word_lsts, words_id):
 
 
 def report_new_words(vocab_path, text_path, output_path):
-    with codecs.open(vocab_path, "r", encoding="utf-8") as f:
-        model = f.read()
-    vocab = set(list(item.split("\t")[1] for item in model.split("\n")))
-
-    with codecs.open(text_path, "r", encoding="utf-8") as f:
-        test = f.read()
-    text = {item.split("\t")[0]: item.split("\t")[1] for item in test.split("\n") if len(item) != 0}
-
-    new_words = {key: value for key, value in text.items() if value not in vocab}
-    save_json(new_words, output_path)
+    t_dataId2word, te_dataId2word = {}, {}
```

Comment:
Nice Work on this function!
I think we don't need a definition here since we are calling the `get_dataId2word_from_path` function, it will return a dictionary for us.